src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -194,28 +201,17 @@ import keys from '../../data/qg-google-keys';
       }
     });
     //As v3 key is used in footer and could also be used on the page with a differnt action, we need to ensure we only load it once
-    if (requireDefaultKey) {
-      //load right away
+    if (loadFooter) {
```

Comment:
it should be only if default is not required and footer exists. since it will break v3 default with footer

Reply:
This looks like working because SWE 4 footer will be there with SWE 4 assets.
This is the testing link for V3 http://qld-gov-au.github.io/qg-web-template/captchaV3.htm